ui/src/containers/CreateVolume.js

```diff
@@ -100,6 +104,7 @@ const CreateVolume = props => {
   const storageClassesName = props.storageClass.map(
     storageClass => storageClass.metadata.name
   );
+  const isSCLoading = useSelector(state => state.app.volumes.isSCLoading);
```

Comment:
The meaning of `SC` may not be obvious to the reader that wasn't involved in the development (myself I was wondering about the meaning for a while).

I suggest doing like l.168 and use `StorageClass` instead of `SC`.